src/SOS/Strike/strike.cpp

```diff
@@ -8722,12 +8723,12 @@ DECLARE_API(ThreadPool)
             accumulatedOffset += offset;
 
             offset = GetValueFieldOffset(vCountsField.MTOfType, W("_data"));
-            if (offset < 0 || FAILED(MOVE(ui64Value, cdaTpInstance + accumulatedOffset + offset)))
```

Comment:
Does this still work with older runtime versions?

Reply:
Yes only the lower 32 bits of the 64-bit value are used, the only change here is to read only those 32 bits. It's ok to read 64 bits too since the field that is read is padded sufficiently.